packages/features/ee/managed-event-types/reassignment/services/ManagedEventManualReassignmentService.ts

```diff
@@ -0,0 +1,882 @@
+              .map((res) => ({
+                type: res.type,
+                error: res.error,
+              })),
+          });
+        }
+      }
+
+      videoCallUrl = evt.videoCallData?.url ?? null;
+
+      if (results.length) {
+        const createdEvent = results[0]?.createdEvent;
+
+        additionalInformation.hangoutLink = createdEvent?.hangoutLink;
+        additionalInformation.conferenceData = createdEvent?.conferenceData;
+        additionalInformation.entryPoints = createdEvent?.entryPoints;
+        evt.additionalInformation = additionalInformation;
+
+        videoCallUrl =
+          additionalInformation.hangoutLink ||
+          createdEvent?.url ||
+          getVideoCallUrlFromCalEvent(evt) ||
+          videoCallUrl;
+      }
+
+      videoCallData = evt.videoCallData;
+
+      try {
+        const responses = {
+          ...(typeof newBooking.responses === "object" && newBooking.responses),
+          location: {
+            value: bookingLocation,
+            optionValue: "",
+          },
+        };
+
+        const finalVideoCallUrlForMetadata = videoCallUrl
+          ? getVideoCallUrlFromCalEvent(evt) || videoCallUrl
+          : null;
+        const bookingMetadataUpdate = finalVideoCallUrlForMetadata
+          ? { videoCallUrl: finalVideoCallUrlForMetadata }
+          : {};
+
+        const referencesToCreateForDb = referencesToCreate.map((reference) => {
+          const { credentialId, ...restReference } = reference;
+          return {
+            ...restReference,
+            ...(credentialId && credentialId > 0 ? { credentialId } : {}),
+          };
+        });
+
+        await this.bookingRepository.updateLocationById({
+          where: { id: newBooking.id },
+          data: {
+            location: bookingLocation,
+            metadata: {
+              ...(typeof newBooking.metadata === "object" && newBooking.metadata
+                ? newBooking.metadata
+                : {}),
+              ...bookingMetadataUpdate,
+            },
+            referencesToCreate: referencesToCreateForDb,
+            responses,
+            iCalSequence: (newBooking.iCalSequence || 0) + 1,
+          },
+        });
+
+        logger.info("Updated booking location and created calendar references", {
+          referencesCount: referencesToCreate.length,
+          videoCallUrl: bookingMetadataUpdate.videoCallUrl ? "[redacted]" : null,
+        });
+      } catch (error) {
+        logger.error("Error updating booking location and references", error);
+      }
+
+      logger.info("Created calendar events for new user");
+    } catch (error) {
+      logger.error("Error creating calendar events for new user", error);
+    }
+
+    return {
+      bookingLocation,
+      videoCallUrl,
+      videoCallData,
+      additionalInformation,
+    };
+  }
+
+  /**
+   * Cancels workflow reminders for the original booking
+   */
+  private async cancelExistingWorkflows({
```

Comment:
Do we really need this separate function? We already do the logging in `ManagedEventWorkflowReminderService.ts`